cmd/kubeadm/app/apis/kubeadm/v1alpha3/bootstraptokenstring.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1alpha2 holds the external kubeadm API types of version v1alpha2
+// Note: This file should be kept in sync with the similar one for the internal API
+// TODO: The BootstrapTokenString object should move out to either k8s.io/client-go or k8s.io/api in the future
```

Comment:
Now is the time to make this change, it really doesn't belong here.

Reply:
This is out of scope for this PR, it isn't worse than what we had with v1alpha1 and v1alpha2 really.
Yes, we're gonna fix this very soon, but it doesn't block all of the other PRs I have lined up for the moment

Reply:
Please file an issue in the kubeadm repo as p1 as it needs to get fixed this cycle. 

/cc @fabriziopandini

Reply:
I added an action item to https://github.com/kubernetes/kubeadm/issues/963 Tracking issue for "Config to v1beta1"